src/net/sourceforge/kolmafia/preferences/Preferences.java

```diff
@@ -671,11 +673,46 @@ private static void loadGlobalPreferences() {
   }
 
   private static void loadUserPreferences(String username) {
-    File file =
+    File userPrefsFile =
         new File(KoLConstants.SETTINGS_LOCATION, Preferences.baseUserName(username) + "_prefs.txt");
-    Preferences.userPropertiesFile = file;
+    File backupFile =
+        new File(KoLConstants.SETTINGS_LOCATION, Preferences.baseUserName(username) + "_prefs.bak");
+    Preferences.userPropertiesFile = userPrefsFile;
 
-    Properties p = Preferences.loadPreferences(file);
+    Properties p = Preferences.loadPreferences(userPrefsFile);
+
+    if (p.size() == 0) {
+      // Something went wrong reading the preferences.
+      if (backupFile.exists()) {
+        KoLmafia.updateDisplay(
+            userPrefsFile + " oould not be read and backup exists, trying backup...");
+        // also tell system out, in case things are really fubar
+        System.out.println("Prefs could not be read and backup exists, trying backup...");
+
+        p = Preferences.loadPreferences(backupFile);
+
+        if (p.size() > 0) {
+          try {
+            java.nio.file.Files.copy(
+                backupFile.toPath(), userPrefsFile.toPath(), StandardCopyOption.REPLACE_EXISTING);
+
+          } catch (IOException ex) {
+            System.err.format("I/O Error when copying file");
+          }
+        }
+      } else {
+        KoLmafia.updateDisplay(
+            userPrefsFile
+                + " could not be read and no backup exists!  Please exit KoLmafia and check your preference files.");
```

Comment:
I suspect you may not want to exit KoLmafia in this case -- or rather, I think you might want to check the directory first. I'm not sure what exiting does, but it might try to save preferences.

Reply:
Updated, please see if this is more helpful.